src/components/Board/Board.jsx

```diff
@@ -26,14 +13,13 @@ const Board = (props) => {
               <div className="row" key={index}>
                 {
                   row.map((tile, index) => {
-                    return (<Tile selected={tile.isSelected} letter={tile.letter} key={index} />)
+                    return (<Tile selected={tile.isSelected} letter={tile} key={index} handleClick={handleClick.bind(this, tile)}/>)
                   })
                 }
-            </div>
+              </div>
```

Comment:
Hmm.. maybe change to 2 character indent to keep it all consistent.  Or use tabs.  Either way, consistency looks nicer.